common-lib/src/main/scala/com/gu/mediaservice/lib/aws/DynamoDB.scala

```diff
@@ -183,6 +187,32 @@ class DynamoDB(credentials: AWSCredentials, region: Region, tableName: String) {
       id, s"REMOVE $key[$index]"
     ) map(j => (j \ key).as[List[T]])
 
+  def objPut[T](id: String, key:String, value: T)
+                 (implicit ex: ExecutionContext, wjs: Writes[T], rjs: Reads[T]): Future[T] = Future {
+
+    val item = new Item().withPrimaryKey(IdKey, id).withJSON(key, Json.toJson(value).toString)
+
+    val spec = new PutItemSpec().withItem(item).withReturnValues(ReturnValue.ALL_OLD)
+    table.putItem(spec)
+    // As PutItem only return the null if the item didn't exist, or the old item if it did,
+    // all we care about is whether it completed.
+  } map (outcome => value)
+
+  def objDelete[T](id: String, key:String, value: T)
+               (implicit ex: ExecutionContext, wjs: Writes[T], rjs: Reads[T]): Future[T] = Future {
+
+    val item = new Item().withPrimaryKey(IdKey, id).withJSON(key, Json.toJson(value).toString)
+
+    val spec = new PutItemSpec().withItem(item).withReturnValues(ReturnValue.ALL_OLD)
+    table.putItem(spec)
```

Comment:
The method wasn't needed as we're using [`deleteItem`](https://github.com/guardian/grid/pull/1613/files#diff-407e9be250074544f6fc2a1f642a4727R63) instead, I suppose I just didn't finish it.